packages/firebase_messaging/firebase_messaging/pubspec.yaml

```diff
@@ -5,28 +5,28 @@ homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/f
 version: 8.0.0-dev.14
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
+  sdk: '>=2.12.0-133.7.beta <3.0.0'
```

Comment:
Migration tool applied these versions; can someone confirm what we're sticking with?

Reply:
That's fine